### DISCUSSION TITLE
why are source files obfuscated and not named `xpui.js` for example

### DISCUSSION THREAD

**Discussion by ohitstom** [Delusoire/bespoke#7] [General]
![image](https://github.com/Delusoire/bespoke/assets/22730962/7610cc1d-56d0-44d2-8b8d-3db94b300b9f)

**Comment by Delusoire** [comment]
Because they're loaded dynamically as a "Blob", I could use a "File" which is a "Blob" with a name but devtools still chooses to show the ObjectURL instead of the "File"'s name.
